src/runtime/profiling.cc

```diff
@@ -228,6 +228,71 @@ String ReportNode::AsCSV() const {
   return s.str();
 }
 
+namespace {
+void print_metric(std::ostream& os, ObjectRef o) {
+  if (o.as<StringObj>()) {
+    os << "\"" << Downcast<String>(o) << "\"";
```

Comment:
should we worry about escaping here?

Reply:
No, metric names should not have quotes.